advanced1/processing.php

```diff
@@ -0,0 +1,163 @@
+<?php
+require 'apicall.php';
+$main = new APIcall('https://www.innoraft.com/jsonapi/node/services');
+$main->call_api();
+$mainResponse = $main->get_data();
+
+/**
+ * Class Field
+ * 
+ * Class of the major sections (services) for the index page.
+ */
+class Field {
+  
+  /**
+   * Stores the title of the field.
+   * 
+   * @var string $title
+   */
+  public $title;
+  
+  /**
+   * Stores the id attribute of the field.
+   * 
+   * @var string $id
+   */
+  public $id;
+  
+  /**
+   * Stores the title of the field in HTML format complete with classes.
+   * 
+   * @var string $displayTitle
+   */
+  public $displayTitle;
+  
+  /**
+   * Stores a list of sub-field links in HTML format.
+   * 
+   * @var string $fieldServices
+   */
+  public $fieldServices;
+  
+  /**
+   * Stores the image link of the field object.
+   * 
+   * @var string $image
+   */
+  public $image;
+  
+  /**
+   * Stores the link of "Explore More" button.
+   * 
+   * @var string $link
+   */
+  public $link;
+  
+  /**
+   * Stores an array of the icons present in the field object.
+   * 
+   * @var array $link
+   */
+  public $icons = [];
+
+  /**
+   * Field constructor function.
+   *
+   * @param string $title The title of the field object.
+   * @param string $id  The id of the field object.
```

Comment:
Mention what this constructor is doing.
Description of the param should be in next line with an indentation.